### ISSUE TITLE
Lactation duct obstruction

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10075]
**Preferred term label:**
Lactation duct obstruction

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Galactocele

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

added as
Mammary blocked duct
HP:0430075
